qemu/tests/virtio_scsi_mq.py

```diff
@@ -38,6 +39,8 @@ def proc_interrupts_results(results):
                 continue
             if cpu_count > 0:
                 irq_key = re.split(":", line)[0]
+                if re.findall(irqs_pattern, re.split(":", line)[-1]):
+                    irqs_key.append(irq_key)
```

Comment:
I think the efficiency can be improved to move the following  code to if re.findall(..) condition.